### PR TITLE
refactoring & decoupling

### PR DESCRIPTION
**Pull request by JaerongA**

### CODE REVIEW

File: aeon/dj_pipeline/streams.py [hunk 1]
```diff
@@ -22,13 +20,21 @@
...
 
-    type: str
-    desc: int
-    streams: tuple
+        device_configs = DeviceConfig.get_configs()
+        device =  device_configs[0]
+        print(f"{device.type}, {device.desc}, {device.streams}")
 
-    _DEVICE_CONFIG = [
+        >> Camera, Camera device, (<function video at 0x7f63ab05bc10>, <function position at 0x7f63ab0083a0>, <function region at 0x7f63ab0088b0>)
```
Comment (ttngu207) [inline aeon/dj_pipeline/streams.py]: Is this a copy/paste mistake?

File: aeon/dj_pipeline/streams.py [hunk 2]
```diff
@@ -22,13 +20,21 @@
 schema = dj.schema(schema_name)
 
 
-class DeviceConfig(NamedTuple):
+class DeviceConfig:
```
Comment (ttngu207) [inline aeon/dj_pipeline/streams.py]: I think we can remove the use of this `DeviceConfig` class completely, as it seems to only serve the purpose of being a namespace

File: aeon/dj_pipeline/streams.py (hunk 1 above)
Comment (JaerongA) [reply aeon/dj_pipeline/streams.py]: It's just an example usage & expected outputs.

File: aeon/dj_pipeline/streams.py (hunk 2 above)
Comment (JaerongA) [reply aeon/dj_pipeline/streams.py]: hmm.. ok let me try & re-structure

**Review by ttngu207 — APPROVED** [review]

**Pull request merged**